docs/framework/wcf/samples/transport-udp.md

```diff
@@ -68,7 +68,7 @@ The UDP Transport sample demonstrates how to implement UDP unicast and multicast
 
 -   The``<xref:System.ServiceModel.Channels.ChannelManagerBase> class implements <xref:System.ServiceModel.Channels.CommunicationObject> and provides a unified base class for <xref:System.ServiceModel.Channels.ChannelFactoryBase> and <xref:System.ServiceModel.Channels.ChannelListenerBase>. The <xref:System.ServiceModel.Channels.ChannelManagerBase> class works in conjunction with <xref:System.ServiceModel.Channels.ChannelBase>, which is a base class that implements <xref:System.ServiceModel.Channels.IChannel>.  
```

Comment:
The \`\` before \<xref:System.ServiceModel.Channels.ChannelManagerBase> should also be removed.